src/seedu/addressbook/parser/Parser.java

```diff
@@ -50,7 +50,7 @@ public Parser() {}
      * @param userInput full user input string
      * @return the command based on the user input
      */
-    public Command parseCommand(String userInput) {
+    public static Command parseCommand(String userInput) {
```

Comment:
Has the call of this function in Main.java been changed to reflect this static method?